xstatic/pkg/font_awesome/__init__.py

```diff
@@ -46,9 +46,4 @@
     # your relative path/filenames should usually be without version
     # information, because either the base dir/url is exactly for this
     # version or the mapping will care for accessing this version.
-    ('cdnjs', 'http'): 'http://cdnjs.cloudflare.com/ajax/libs/font-awesome/%s/' % VERSION,
-    ('cdnjs', 'https'): 'https://cdnjs.cloudflare.com/ajax/libs/font-awesome/%s/' % VERSION,
```

Comment:
https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css this is from first search result for "fontawesome cdn".

Reply:
I could add the min.css, but there are now lots more, 
https://cdnjs.com/libraries/font-awesome

tree -L 1
.
├── css
├── js
├── less
├── LICENSE.txt
├── metadata
├── scss
├── sprites
├── svgs
└── webfonts

The license file is included in the package from 
https://use.fontawesome.com/releases/v6.2.1/fontawesome-free-6.2.1-web.zip

Reply:
The CDN entries only give the base URL (same as the local path is only giving the starting path). The rest of the path/filename has to be added by the user.

Reply:
thx for clarifying